tests/test_sync.py

```diff
@@ -1,28 +1,29 @@
 #!/usr/bin/env python
 # coding: utf-8
 
-from __future__ import (
-    absolute_import,
-    division,
-    print_function,
-    unicode_literals,
-)
-
+import base64
 import datetime
+import io
 import logging
 import os
 import sys
+import json
+import hashlib
 import unittest
 import uuid
+from argparse import Namespace
 
 import boto3
+from botocore.vendored import requests
```

Comment:
I moved this line to the end of this block, but for my education, can you give the rationale? I'd like to group boto-related imports together...